client/task-list/task-list.js

```diff
@@ -246,6 +247,26 @@ export const TaskList = ( {
 		return <div className="woocommerce-task-dashboard__container"></div>;
 	}
 
+	const expandLabel = sprintf(
+		/* translators: %i = number of hidden tasks */
+		__( 'Show %i more tasks.', 'woocommerce-admin' ),
```

Comment:
<img width="547" alt="Screen Shot 2021-05-03 at 11 29 11 AM" src="https://user-images.githubusercontent.com/63922/116897427-79460780-abf2-11eb-9024-e3b25e3131c1.png">

We can properly handle the single/plural using `_n()`.